index.html

```diff
@@ -2824,8 +2879,31 @@ <h4 id="anyPass"><code><a href="https://github.com/sanctuary-js/sanctuary/blob/v
 
 <a class="pilcrow h3" href="#array">¶</a>
 <h3 id="array">Array</h3>
+<a class="pilcrow h4" href="#array">¶</a>
+<h4 id="array"><code><a href="https://github.com/sanctuary-js/sanctuary/blob/v1.0.0/index.js#L2950">array</a> :: b -> (a -> <a href="https://github.com/sanctuary-js/sanctuary-def/tree/v0.19.0#Array">Array</a> a -> b) -> <a href="https://github.com/sanctuary-js/sanctuary-def/tree/v0.19.0#Array">Array</a> a -> b</code></h4>
```

Comment:
`id="array"` appears twice in the document. Fortunately the targets are in close proximity, so the conflict won't cause serious problems. Regardless, we should fix this at some point, perhaps by changing __Array__ to __Foldable__.